packages/mcp/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@hono/mcp",
+  "version": "0.1.0",
```

Comment:
Can you set it to `0.0.0` since the changeset will bump the version number automatically.